extensions/template/README.md

```diff
@@ -1,53 +1,60 @@
+    - On a self-hosted Sourcegraph instance, select **User menu > Extensions**, search for `sourcegraph/LANGID`, and enable it.
+1. Visit any LANG code file on Sourcegraph.
 1. Hover over a token in the code file.
-    - Click **Find references** to see all references to the token.
+    - See a description of the token.
     - Click **Go to definition** (if available) to go to the token's definition.
-
-### Optional features
-
-The command palette (<kbd>≡</kbd>) lets you toggle other options:
-
--   **Basic code intel: Enable fuzzy hovers** — to see hovers and docstrings (recommended, will become default soon)
--   **Basic code intel: Enable cross-repository definitions** — to find definitions across repositories, which increases the risk of incorrect definition results
--   **Basic code intel: Disable fuzzy def/ref matching** — to disable "Go to definition" and "Find references" results from the extension (easier than enabling/disabling the extension itself if you frequently need to toggle it, such as if you sometimes want to see only precise results from language-specific extensions)
+    - Click **Find references** to see all references to the token.
 
 ### On your code host
 
 This extension adds the same features to code files and diffs on your code host if you're using the [Sourcegraph browser extension](https://docs.sourcegraph.com/integration/browser_extension). To use it on your code host:
 
-1. Follow the [usage steps](#usage) above to enable this `sourcegraph/LANGNAME` extension.
+1. Follow the [usage steps](#usage) above to enable this extension.
 1. Install the [Sourcegraph browser extension](https://docs.sourcegraph.com/integration/browser_extension).
     - If you're using it with a self-hosted Sourcegraph instance, enter the Sourcegraph instance URL into the Sourcegraph browser extension options menu. Then click the gear icon and enable _Experimental features: Use extensions_.
-1. Visit any file on your code host and hover over a token to see **Find references** and (if available) **Go to definition** actions.
+1. Visit any file on your code host and hover over a token to see a description of the token, a **Go to definition** action (if available), and a **Find references** action.
 
-![screenshot of using LANGNAME on GitHub](https://user-images.githubusercontent.com/1976/50882271-0c5f3180-1399-11e9-9697-e4e4fa4e29e9.png)
+![screenshot of using LANGID on GitHub](https://user-images.githubusercontent.com/1976/50882271-0c5f3180-1399-11e9-9697-e4e4fa4e29e9.png)
 
-### Limitations
+## Basic code intelligence
 
-Because this extension uses text-based heuristics, its definition and reference results are not precise:
+This extension comes with built-in code intelligence provided by [search-based heuristics](https://docs.sourcegraph.com/user/code_intelligence/basic_code_intelligence). Because this extension uses text-based heuristics, its definition and reference results may not be precise:
```

Comment:
I think "are not" was better: They may be _correct_ (for a given query) but they are not _precise_.

Reply:
Updated in all three places.